project-reports/2023/2023-Q1-Hyperledger-Besu.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: 2023 Q1 Hyperledger Besu
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+Besu is having a big boost in activity on a number of metrics. Besu is climbing back its network share on Ethereum Mainnet and hovering around 10%. We now have new contributors from Web3Labs taking on issues and providing private network support. This adds another organization alongside existing contributors. The Discord is very active and monitored by a number of teams. Contributor calls are productive and grants are being distributed equitably. A series of Besu related workshops should also bring in new contributors working alongside the TOC and HLF leadership. We hope to onboard at least one of the banks or FinTechs we chatted with.
+
+Additionally, with Shanghai enabling Ether withdrawals from validators, HLF can begin to use some of these funds to pay for project infrastructure and hopefully some incentives for community engagement down the line alongside CSI and other maintainer groups. 
+
+# Required Information
+All completed for Besu. 
+
+- Have you switched from master to main in all your repos?
+- Have you implemented the [Common Repository Structure](../guidelines/repository-structure.md) in all your repos?
+- Has your project implemented these inclusive language changes listed below to your repo? You can optionally [use the DCI Lint tool](https://github.com/petermetz/gh-action-dci-lint#usage) to make this a recurring action on your repo.
+  - master → main
+  - slave → replicas
+  - blacklist → denylist
+  - whitelist → allowlist
+- Have you added an [Inclusive Language Statement](https://wiki.hyperledger.org/display/TSC/Inclusive+Language+Example) to your project's documentation and/or Wiki pages?
+
+# Questions/Issues for the TSC
+
+Will the TSC deliver a unified CI/CD strategy across HLF projects to avoid ambiguity going forward? 
```

Comment:
We have been talking about CI/CD and GitHub actions in the past couple of TOC meetings. Can you comment on the ambiguity that you are seeing and what it is you would like to see in a unified strategy? Not in the report, but just as a thread in this PR.

Reply:
Frankly, it was the willingness to provide no transparency around the CI and then the expectation Consensys would pay indefinitely without a framework in place from Hyperledger. Also the deprecation of CI for GHA that never had a documented plan or communication up front and the sour taste from all of that back and forth only to end up footing the bill. This will be different specifically for Besu as we use the Client Incentive Program funds to pay for the CI going forward. But literally any policy or discussion would be better than what happened.